### PR TITLE
Add TEF logo to intro. Bug 1119435.

### PR DESCRIPTION
**Pull request by jpetto**
- Organize hello files

### CODE REVIEW

File: media/css/firefox/hello/index.less [hunk 1]
```diff
@@ -200,6 +200,21 @@ https://bugzilla.mozilla.org/show_bug.cgi?id=1113917#c4
     background: url(/media/img/firefox/hello/hello-intro.png) center center no-repeat;
 }
 
+#powered-by {
+    .font-size(12px);
```
Comment (alexgibson) [inline media/css/firefox/hello/index.less]: Nit: `@smallFontSize`

File: bedrock/firefox/templates/firefox/hello/index.html [hunk 2]
```diff
@@ -117,6 +119,10 @@ <h2 class="heading-tertiary">
 
       <button class="try-hello" id="try-hello-intro">{{ _('Try Firefox Hello') }}</button>
 
+      <aside id="powered-by">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/hello/index.html]: Does this need to be in an l10n tag, or do we already have these strings translated?

Comment (flodolo) [inline bedrock/firefox/templates/firefox/hello/index.html]: We've imported strings from product, no need for `l10n_has_tag`

Comment (alexgibson) [inline bedrock/firefox/templates/firefox/hello/index.html]: Thanks @flodolo!

**Comment by alexgibson** [comment]
One tiny nit, but otherwise it's an r+wc from me

+1 to reorganizing the assets into the `/firefox/hello` directory :+1:

**Comment by jpetto** [comment]
Thanks for the review @alexgibson! All should be good to go as soon as L10n is ready.

**Comment by jpetto** [comment]
Waiting for confirmation from Arcadio that we can merge a couple days ahead of schedule.

**Comment by ckprice** [comment]
@jpetto confirmation received. Good to merge.

**Pull request merged**